.circleci/config.yml

```diff
@@ -31,24 +84,39 @@ jobs:
 
     steps:
       - checkout
-      - run: bundle install
-      - run: yarn install
-      - run: git config --global user.email "ci-build@coopdevs.org"
-      - run: git config --global user.name "CI"
-      - add_ssh_keys:
-          fingerprints:
-            - "f6:8f:6b:93:bf:a3:a1:52:46:c6:2b:54:b6:5c:2e:23"
-      - run: git config -l
-      - run: ./deploy
+      - *restore_bundler_cache
+      - *bundle_install
+      - *save_bundler_cache
+      - *restore_yarn_cache
+      - *yarn_install
+      - *save_yarn_cache
+      - *git_config
+      - *add_ssh_keys
+      - *git_config_l
+      - *deploy
+
 
 workflows:
   version: 2
-  build-and-deploy:
+
+  feature:
+    jobs:
+      - build:
+          <<: *feature_branch_filter
+  develop:
```

Comment:
so if I understand correctly, you suggest to split the existing `build-and-deploy` workflow into two: `feature` and `develop`?

Reply:
sort of. i'm still keeping a successful build as a required step for deployment. this way you can run build, and eventually lint or test jobs just for the feature branches, and the deployment gets separated by environment. so if you have a separate development URL, you can have deployments to `develop` go to that site and have production deployments build to a different URL.